help/compute/diff-studio.md

```diff
@@ -171,18 +168,32 @@ You can set new values for parameters and change values for functions.
 
 ![Multi-stage model - loop](pics/DiffStudio-loop.gif)
 
-Apply the `#update` feature to obtain acyclic simulation.
-Set the length of the new modeling stage in the mandatory `length` variable.
-Inside you can use any mathematical expression to modify functions and parameters,
-same as for the `#loop` section.
+### Multi-stage model
+
+The multi-stage model is a model where parameters and function values can vary for different
```

Comment:
Simplify this.

Reply:
Something like that:

In multistage model, parameters might change between different argument ranges (stages).